framework/console/controllers/CacheController.php

```diff
@@ -270,6 +270,12 @@ private function findCaches(array $cachesNames = [])
                 $caches[$name] = $component['class'];
             } elseif (is_string($component) && $this->isCacheClass($component)) {
                 $caches[$name] = $component;
+            } elseif ($component instanceof \Closure) {
+                $cache = Yii::$app->get($name);
```

Comment:
this is weird. $component does not seem to be used? Can this be supplemented with a `comment`?

Reply:
Why should it be used? It's type is checked then it's being instantiated by name.